toolkit/tools/pkggen/worker/create_worker_chroot.sh

```diff
@@ -44,12 +46,16 @@ install_one_toolchain_rpm () {
     fi
 }
 
+$unmount_tool "$chroot_builder_folder" | tee -a "$chroot_log"
 rm -rf "$chroot_builder_folder"
 rm -f "$chroot_archive"
 rm -f "$chroot_log"
 mkdir -p "$chroot_builder_folder"
 mkdir -p "$log_path"
 
+# Create tmpfs in chroot_builder_folder
+mount -t tmpfs -o size=1500M tmpfs "$chroot_builder_folder"
```

Comment:
Is this safe to do in an unprivileged container? Or are there no scenarios in which the worker chroot needs to be buildable in a container environment?